internal/runners/packages/searchView.go

```diff
@@ -0,0 +1,134 @@
+package packages
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/charmbracelet/bubbles/viewport"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+	"golang.org/x/term"
+)
+
+const (
+	verticalMargin = 7
+)
+
+type errMsg error
+
+type view struct {
+	width        int
+	height       int
+	content      string
+	remaining    int
+	packageNames []string
+	ready        bool
+	err          error
+	viewport     viewport.Model
+}
+
+func NewView() (*view, error) {
+	width, height, err := term.GetSize(int(os.Stdout.Fd()))
```

Comment:
Don't use `os` directly, instead use `output.Config().OutWriter`.

Reply:
This is expecting a file descriptor. The `OutWriter` is an `io.Writer` interface.

Reply:
Can we add the `Fd()` method on `Outwriter`? I'm not confident that we'll end up using this long term but having everything agree on a same source of truth will make refactoring this later easier.

Reply:
Added a method to the output `Config` type.